src/collection.jl

```diff
@@ -9,6 +9,16 @@ end
 
 parse_name(filename, ext, ::Val{true}) = filename
 
+
+# transposes the data and saves it, the reason is same as that for getdata
+function setdata(file_path, data)
```

Comment:
Not sure the name of the function is super clear about what it does?  Probably a docstring would also be in order here

Reply:
Should we expose this function to the user?

Reply:
`setdata` is a natural opposite to `getdata`, but unfortunately is not really accurate for what we're doing (creating a new file and writing data to it).

I would recommend a name closer to `write_data` or something that takes a path and data. Maybe adding an overwrite option?

I wouldn't expose this by default (same as `getdata`), but I would keep it well-documented. This is a function I can see being used for convenience, but I don't want it to be part of the public API because it is just some small functionality that we ought to be able to change when/how we want.